docs/sources/architecture/distributor.md

```diff
@@ -6,6 +6,9 @@ weight: 1000
 
 This document builds upon the information in the [Loki Architecture](./) page.
 
+Distributors communicate with ingesters via [gRPC](https://grpc.io). They are
+stateless and the quantity of ingesters can be increased or decreased as needed.
```

Comment:
```suggestion
Distributors are stateless and communicate with ingesters via [gRPC](https://grpc.io). The quantity of distributors can be increased or decreased as needed.
```
This sentence confuses me a bit. Distributors are indeed stateless, while ingesters are stateful. The quantities of both can be changed, but it is much easier to do so with the stateless distributors.